specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json

```diff
@@ -1718,12 +1758,6 @@
     "TranscriptConversationItem": {
       "type": "object",
       "description": "Additional properties for supporting transcript conversation.",
-      "required": [
-        "text",
-        "lexical",
-        "itn",
-        "maskedItn"
-      ],
```

Comment:
How is this change related, and are these not still required? Even for response objects, we want a list of never-null/missing properties for better code generation. Was this intentional, or just a mistake? I.e. I don't mind unrelated changes in the PR, it just isn't obvious if this was intended.